LICENSE_AGPL.txt

```diff
@@ -1,7 +1,7 @@
                     GNU AFFERO GENERAL PUBLIC LICENSE
                        Version 3, 19 November 2007
 
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
```

Comment:
@kspearrin I'm hesitant to approve this change. The [html version](https://www.gnu.org/licenses/agpl-3.0.en.html) of the license does have an https link but I don't want to go around approving license file changes without permission.